src/Commands/BuildToolsCommand.php

```diff
@@ -658,9 +659,13 @@ protected function expandSourceAliases($source)
      */
     protected function autodetectUpstream($siteDir)
     {
-        return 'Empty Upstream';
-        // or 'Drupal 7' or 'WordPress'
-        // return 'Drupal 8';
+        if (file_exists($siteDir . '/wp-cli.yml')) {
```

Comment:
The Composer-based way of doing WordPress moves `wp-admin` too. In our example it is `web/wp/wp-admin`. I don't think we can assume that. I'm leaning toward `web/wp-config.php`. @ataylorme, do you have thoughts on what file would be the best to check for to determine if a repo is WordPress?